sdcm/nemesis.py

```diff
@@ -546,6 +552,21 @@ def disrupt_nodetool_cleanup(self):
             for keyspace in test_keyspaces:
                 node.run_nodetool(sub_cmd="cleanup", args=keyspace)
 
+    def _prepare_test_table(self, ks='keyspace1'):
```

Comment:
why do you use C-S to pre-create schema and not `longevity_test.LongevityTest#_pre_create_schema`?

Reply:
In this special case, set pre_create_schema can solve the problem. keyspace1.standard1 (with default schema) will be created.

pre_create_schema can only created keyspace1.standard1 with default 5 column.
The workloads will fail if column size is assigned in c-s cmdline.

eg: `-col 'n=FIXED(10) size=FIXED(512)'`